### ISSUE TITLE
add a CI pipeline to test accessibility of the HTML publisher on deployment

### ISSUE THREAD

**Issue by mahdikhashan** [boudicca-events/boudicca.events#325]
related issues:

1. #294 
  1.1 A11Y needs that should be considered
  1.2 information / tags that should be added / enriched

2. #293 
  2.1 A11Y for blind users
